### PR TITLE
Add docs for homewizard_climate

### PR DESCRIPTION
**Pull request by mepla**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- [Brands PR](https://github.com/home-assistant/brands/pull/3938)
- [New integration PR](https://github.com/home-assistant/core/pull/84105)

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/homewizard_climate.markdown [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  
+This integration allows you to control your Homewizard Climate devices in Home Assistant. There are a multitude of brands that use Homewizard apps for their smart controls.
+
+  
+### Supported Devices
+This integration is in an early stage of development and only works for the following device types returned from the Homewizard Climate API:
+
+- `heaterfan`
+
+It has been tested on the following devices (even though it might work on others too):
```
Comment (DCSBL) [inline source/_integrations/homewizard_climate.markdown]:
```suggestion
It has been tested on the following devices:
```

Your API implementation allows only the heater fan, none other will work.

**Review by DCSBL — COMMENTED** [review]
Nice work!

I think we should mention that you are using an undocumented/'private' api. It should continue to work, but support can be removed at any time without notice.

**Comment by mepla** [comment]
@DCSBL Makes sense, added a warning/disclaimer at the end.

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**